src/index.js

```diff
@@ -2,7 +2,7 @@ import apiCreate from './api/create';
 import apiEmit from './api/emit';
 import apiFragment from './api/fragment';
 import apiInit from './api/init';
-import apiProperties from './api/properties/index';
```

Comment:
I use Rollup now instead of JSPM where #346 was giving me issues, but I think removing the explicit `index` would trigger it again for JSPM users (the SystemJS loader doesn't support the implied `index.js` import)